### PR TITLE
For review

### PR DESCRIPTION
**Pull request by bakunyo**
レビュー会向け

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -2,7 +2,14 @@
...
+if Rails.env.development?
+  user = FactoryGirl.create(:user, email: 'test@example.com', password: 'password')
+  FactoryGirl.create(:user_role, :admin, user: user)
 
-5.times { FactoryGirl.create(:monthly_report_tag) }
+  FactoryGirl.create(:user, email: 'testuser@example.com', password: 'password')
+
+  5.times { FactoryGirl.create(:monthly_report_tag) }
+else
+  user = User.create(name: 'admin', email: 'test@example.com', password: 'password', employee_code: 'x', entry_date: Date.today, beginner_flg: true)
```
Comment (freezer530000) [inline db/seeds.rb]: **[rubocop]**
- Line is too long. [148/128] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: spec/requests/monthly_report_spec.rb [hunk 2]
```diff
@@ -10,6 +10,12 @@
     it { expect(response.body).to match report.user.name }
   end
 
+  describe "root_path" do
```
Comment (freezer530000) [inline spec/requests/monthly_report_spec.rb]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

**Comment by bakunyo** [comment]
レビュー会向けに作ったつもりだったけど、このままコードレビュー＆マージのいつものフローでお願いします！

**Comment by Nobuhiko-Kudo** [comment]
:ok_woman:

**Comment by ta-chibana** [comment]
:elephant:

**Pull request merged**